### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ss1208**
I have suggested few changes in README.md.
Kindly review.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info https://github.com/kubernetes/website/pull/29431#issuecomment-900036615

**Comment by ss1208** [comment]
Signed the CLA ........

File: README.md [hunk 1]
```diff
@@ -178,10 +178,10 @@ For more information about contributing to the Kubernetes documentation, see:
 | [Italian](README-it.md)    | [Ukrainian](README-uk.md)  |
 | [Japanese](README-ja.md)   | [Vietnamese](README-vi.md) |
 
-## Code of conduct
+## Code of Conduct
```
Comment (reylejano) [inline README.md:181]: Please refer to the[ Kubernetes style guide](https://kubernetes.io/docs/contribute/style/style-guide/#headings), sentence case is used for headings
```suggestion
## Code of conduct
```
Please revert

File: README.md [hunk 2]
```diff
@@ -178,10 +178,10 @@ For more information about contributing to the Kubernetes documentation, see:
...
 
 Participation in the Kubernetes community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 
-## Thank you
+## Vote of Thanks
```
Comment (reylejano) [inline README.md:185]: I prefer the original and see my [comment](https://github.com/kubernetes/website/pull/29431/files#r703961820) earlier about capitalization in headings
```suggestion
## Vote of Thanks
```

File: README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ git submodule update --init --recursive --depth 1
 
 ## Running the website using a container
 
-To build the site in a container, run the following to build the container image and run it:
+In order to build the site in a container, run the following to build the container image:
```
Comment (reylejano) [inline README.md:39]: See the [Kubernetes style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language) about using simple and direct language
```suggestion
In order to build the site in a container, run the following to build the container image:
```
With the example in the style guide, I prefer the original

**Comment by reylejano** [comment]
Hi @ss1208 , please sign the [CLA](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement) for this PR to be reviewed.
A PR wrangler may close this PR within 7 days if the CLA has not been signed. You can re-open the PR when the CLA has been signed

**Comment by reylejano** [comment]
Hi @ss1208 , since there has been no action to sign the CLA, I am closing this PR
Feel free to re-open the PR when you have signed the CLA
/close

**Pull request closed without merging**